### ISSUE TITLE
Copy scripts and stylesheets into user set directory

### ISSUE THREAD

**Issue by ThomasDavisonsGit** [DigitalCommons/goth#11]
There are 3 scripts/stylesheets within the ``assets`` folder. These are needed to make the html look pretty, the html expects to find them in the same directory as itself. Currently, these 3 files are copied to the current working directory when GOTH is run. This is done by ``generator.rb``'s ``copy_assets`` method. 

Task: add a command line option to set this output directory to something other that ``Dir.pwd`` as set by the user.

**Comment by wu-lee** [comment]
Of the three scripts:
- `bootstrap.min.css` can be converted to a link to a content-delivery network hosting this asset
- `bootstrap.min.js` is unused and can be removed
- `jquery.js` is unused and can be removed

Additionally, there is a link in the template to `codemirror.css`, which is a local link, but there is no corresponding file.  This is therefore broken and not adding anything, so can be removed too.

**Comment by wu-lee** [comment]
I've converted the `bootstrap.min.css` local link into a CDN link to the same version of bootstrap (which is really old, but we can tinker with that later)

I've removed all the other assets and links, which are unused or broken.

Therefore, there are no assets to copy any more, so I've also removed the code to copy these assets.

This solves the problem outlined in this ticket, although not in the way originally described.
